### PR TITLE
M365 pipeline conversion for this repo

### PR DESCRIPTION
**Pull request by rohandubal**

### CODE REVIEW

File: .azuredevops/build-ci.yml [hunk 1]
```diff
@@ -12,49 +12,77 @@ pr:
...
+  repositories:
+    - repository: m365Pipelines
+      type: git
+      name: 1ESPipelineTemplates/M365GPT
+      ref: refs/tags/release
+extends:
+  template: v1/M365.Official.PipelineTemplate.yml@m365Pipelines
+  parameters:
+    pool:
+      name: NewLarge-linux-1ES
```
Comment (alexvy86) [inline .azuredevops/build-ci.yml:25]: I think the fact that this pipeline is trying to use a pool that doesn't exist today points to how little activity we see/expect on this repo. The pipeline is even disabled in the internal project. And the name of that non-existent pipeline suggests to me this doesn't need the Large pool.
```suggestion
      name: Small-1ES
```

**Review by alexvy86 — APPROVED** [review]
One comment below, otherwise lgtm. As in the comment, I'll note that this pipeline will only run for PRs, it's disabled in the internal project.

**Comment by alexvy86** [comment]
Ok I was wondering why the PR didn't trigger the pipeline, and neither did my manual attempt at getting it to trigger. The pipeline is configured in ADO to not run for PRs from forks, so I think we want to do this in a branch within the microsoft/FluidStorybook repo.

**Comment by alexvy86** [comment]
Alternatively... @tylerbutler you seem to be the one who last touched this repo in 2021. Is it in use at all? Should we disable the pipeline altogether instead of trying to update it? (And maybe archive the repo?)

**Comment by tylerbutler** [comment]
This repo hasn't been used since 2021 or so. We can probably archive it.

**Comment by alexvy86** [comment]
@rohandubal let me know if you feel different, but based on the conversation above I'll close this PR.

**Pull request closed without merging**